index.html

```diff
@@ -95,12 +95,12 @@ <h1 class="intro">
       </div>
 
       <div class="project-section project2">
-        <img class="project-img" src="images/SnapshootPortfolio(1).jpg" alt="">
-        <img class="project-img-desktop img2" src="images/desktop-images/img2.png" alt="">
```

Comment:
- [ ] kindly add alt description to all your images. it help to know what the image is about.

Reply:
Yes it is added this is the changed code that's why it is in red @tufoinnkuo10, the only place where it is not added is in the images used to style the page like the circles and arrows